data/core/units/humans/Mage.cfg

```diff
@@ -20,7 +20,8 @@
 
 It is an irony that, with all their knowledge, and their unassuming monopoly thereof, the collective community of magi could likely rule society, were they ever to try. However, their true love is neither money, nor power, and those who see the study of magic as a means to such ends often lack the very conviction required for true mastery.
 
-Physically frail, and lacking familiarity with combat, magi do possess certain arts which are of great utility in battle."+{SPECIAL_NOTES}+{SPECIAL_NOTES_MAGICAL}
+Physically frail, and lacking familiarity with combat, magi do possess certain arts which are of great utility in battle."
+    {SPECIAL_NOTE_MAGICAL}
```

Comment:
This is definitely an improvement over the previous system, and I support it as it stands. However, I wonder if it's possible to embed the `[special_note]` use into the ABILITY_CURES macro directly, as in `[ability] id=cures special_note="..." [/ability]`? (Or maybe a subtag rather than a key) This way, it won't be possible to have just one of SPECIAL_NOTES_CURES/ABILITY_CURES without the other (superseding the wmllint checks).

Reply:
I do want to do that, actually. That wouldn't entirely obsolete this system however, since not all special notes are connected to a special ability.

I was thinking I could merge this and then go on to add `special_note=` to abilities, but would you prefer I just do that in this PR instead?

Reply:
I don't mind whether the further change will be done in this PR or separately. Thanks for asking.